src/components/UserInformation/UserInformation.types.ts

```diff
@@ -0,0 +1,12 @@
+import { Dispatch } from 'redux'
+import { CallHistoryMethodAction } from 'connected-react-router'
+import { UserInformationComponentProps } from 'decentraland-ui'
+
+export type Props = Partial<UserInformationComponentProps> & {
+  onClickMyAssets: () => void
+  onClickMyLists: () => void
+}
+
+export type MapStateProps = Pick<Props, 'isSignedIn' | 'isSigningIn' | 'hasActivity'>
+export type MapDispatchProps = Pick<Props, 'onClickActivity' | 'onClickSettings' | 'onClickMyAssets' | 'onClickMyLists' | 'onSignOut'>
```

Comment:
'onClickActivity' & 'onClickSettings' are not defined in the container. That's why the build is failing.